editors/code/src/commands.ts

```diff
@@ -152,7 +152,7 @@ export function onEnter(ctx: Ctx): Cmd {
     }
 
     return async () => {
-        if (await handleKeypress()) return;
+        if (ctx.config.enhancedEnter && await handleKeypress()) return;
```

Comment:
Could you wrap this in a `try` block with `default:type` being performed on an exception, like when the server crashed?

Reply:
That should already be the current behavior due to the `.catch` call in `handleKeypress`